hudi-hadoop-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -202,6 +202,11 @@ public void addPropsFromStream(BufferedReader reader, StoragePath cfgFilePath) t
     }
   }
 
+  @Override
+  public TypedProperties getGlobalProperties() {
```

Comment:
Move the logic of `getGlobalProps()` directly into this new method?

Reply:
it was used a number of times. And we can't override static method